src/sentry/runner/commands/backup.py

```diff
@@ -2,54 +2,198 @@
+    "--merge_users",
+    default=False,
+    is_flag=True,
+    help=MERGE_USERS_HELP,
+)
 @click.option("--silent", "-q", default=False, is_flag=True, help="Silence all debug output.")
 @configuration
-def import_(src, silent):
-    """Imports core data for a Sentry installation."""
+def import_users(src, filter_usernames, merge_users, silent):
+    """
+    Import the Sentry users from an exported JSON file.
+    """
 
-    _import(
+    import_in_user_scope(
         src,
-        ImportScope.Global,
-        OldImportConfig(
-            use_update_instead_of_create=True,
-            use_natural_foreign_keys=True,
-        ),
+        flags=ImportFlags(merge_users=merge_users),
+        user_filter=parse_filter_arg(filter_usernames),
         printer=(lambda *args, **kwargs: None) if silent else click.echo,
     )
 
 
-@click.command()
+@import_.command(name="organizations")
+@click.argument("src", type=click.File("rb"))
+@click.option(
+    "--merge_users",
+    default=False,
+    is_flag=True,
+    help=MERGE_USERS_HELP,
+)
+@click.option(
+    "--filter_org_slugs",
+    default="",
+    type=str,
+    help="An optional array of organizations to include. "
+    "If this option is not set, all encountered organizations are imported. "
+    "Users not members of at least one organization in this set will not be imported.",
+)
+@click.option("--silent", "-q", default=False, is_flag=True, help="Silence all debug output.")
+@configuration
+def import_organizations(src, filter_org_slugs, merge_users, silent):
+    """
+    Import the Sentry organizations, and all constituent Sentry users, from an exported JSON file.
+    """
+
+    import_in_organization_scope(
+        src,
+        flags=ImportFlags(merge_users=merge_users),
+        org_filter=parse_filter_arg(filter_org_slugs),
+        printer=(lambda *args, **kwargs: None) if silent else click.echo,
+    )
+
+
+@import_.command(name="global")
+@click.argument("src", type=click.File("rb"))
+@click.option("--silent", "-q", default=False, is_flag=True, help="Silence all debug output.")
+@configuration
+def import_global(src, silent):
+    """
+    Import all Sentry data from an exported JSON file.
+    """
+
+    import_in_global_scope(
+        src,
+        printer=(lambda *args, **kwargs: None) if silent else click.echo,
+    )
+
+
+@click.group(name="export")
+def export():
+    """Exports core data for the Sentry installation."""
+
+
+@export.command(name="users")
 @click.argument("dest", default="-", type=click.File("w"))
 @click.option("--silent", "-q", default=False, is_flag=True, help="Silence all debug output.")
 @click.option(
-    "--indent", default=2, help="Number of spaces to indent for the JSON output. (default: 2)"
+    "--indent",
+    default=2,
+    type=int,
+    help="Number of spaces to indent for the JSON output. (default: 2)",
+)
+@click.option(
+    "--filter_usernames",
+    default="",
+    type=str,
+    help="An optional array of users to include. "
```

Comment:
same as below, might be good to add that this is in the form of a comma separated string

Reply:
Done.